packages/labs/gen-wrapper-react/src/index.ts

```diff
@@ -94,8 +94,8 @@ const packageJsonTemplate = (
       },
       peerDependencies: {
         // TODO(kschaaf): make react version(s) configurable?
-        react: '^17 || ^18',
-        '@types/react': '^17 || ^18',
```

Comment:
Can these go back to 17 || 18 now? Or does it break if we do that? Seems best for the wrapper to support wider versions if possible.

Reply:
let's try it out!

Reply:
success!